docs/source/user-guide/installation.md

```diff
@@ -28,9 +28,15 @@ Make sure the following requirements are met and software installed on your mach
 
 ## Requirements
 
-- [Apache Spark supported by Comet](overview.md#supported-apache-spark-versions)
-- JDK 8 and up
-- GLIBC 2.17 (Centos 7) and up
+### Supported Apache Spark Versions
+For more details, refer to the documentation: [Apache Spark supported by Comet](overview.md#supported-apache-spark-versions)
+
+- **Apache Spark 3.3, 3.4, or 3.5**
```

Comment:
We would like to remove duplications in doc for Supported Apache Versions @adi-kmt Is there any specific reason of having this duplication?

Reply:
Main motive was to indicate the different jdk versions wrt to the apache spark versions. Do you think it could be written in a better way?